xcodeproj/internal/xcschemes/xcschemes.bzl

```diff
@@ -1189,7 +1191,27 @@ Address Sanitizer cannot be used together with Thread Sanitizer.
         ),
     )
 
-def _autogeneration_config(scheme_name_exclude_patterns = None):
+def _options(
```

Comment:
Since these are specific to tests (and there could be/will be one for "run" as well), can you name this `_test_options` here and elsewhere?